node/src/components/rpc_server/rpcs/common.rs

```diff
@@ -77,35 +77,34 @@ pub(super) async fn run_query_and_encode<REv: ReactorEventT>(
 #[serde(deny_unknown_fields)]
 #[serde(rename_all = "snake_case")]
 pub enum ErrorData {
-    /// The lowest block height from which this node has contiguous blocks to the tip of the chain.
-    LowestContiguousBlockHeight(u64),
+    /// The highest contiguous height range of fully available blocks.
```

Comment:
I think we should add that the range is inclusive to the docs here, as that will carry over to the description provided in the JSON-RPC schema.

Reply:
Comment updated in e748d51648d38e1362794622919a322a76537882